tests/pam_duo-4.t

```diff
@@ -96,3 +96,20 @@ Test getting hostname
   correct hostname
   [1]
 
+Test max prompts equals one
+  $ ./testpam.py -d -c confs/mockduo_autopush.conf -f pam_prompt true
```

Comment:
for clarity do you think you could just copy this config file and rename it to mockduo_prompts_1.conf.

And them maybe rename `mockduo_prompts.confg` to `mockduo_prompts_default.conf`?
I think it will help a reader of the test track what is happening better.